### PR TITLE
docs: add doc for logging

### PR DESCRIPTION
**Pull request by Lou1415926**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: site/content/docs/include/common-svc-fields.en.md [hunk 1]
```diff
@@ -215,5 +215,16 @@ Optional. Defaults to `""`. The ID of the EFS access point to connect to. If usi
 
 <div class="separator"></div>
 
+<a id="logging" href="#logging" class="field">`logging`</a> <span class="type">Map</span>  
+The logging section contains parameters for log configuration options to send to a firelens log driver for the container.
```
Comment (huanjani) [inline site/content/docs/include/common-svc-fields.en.md]:
```suggestion
The logging section contains log configuration parameters for your container's FireLens log driver.
```

File: site/content/docs/manifest/scheduled-job.en.md [hunk 2]
```diff
@@ -222,6 +222,19 @@ Optional. Defaults to `true`. Whether or not to use IAM authorization to determi
 <span class="parent-field">volume.efs.auth.</span><a id="access_point_id" href="#access-point-id" class="field">`access_point_id`</a> <span class="type">String</span>  
 Optional. Defaults to `""`. The ID of the EFS access point to connect to. If using an access point, `root_dir` must be either empty or `/` and `auth.iam` must be `true`.
 
+<div class="separator"></div>
+
+<a id="logging" href="#logging" class="field">`logging`</a> <span class="type">Map</span>  
+The logging section contains parameters for log configuration options to send to a firelens log driver for the container.
```
Comment (huanjani) [inline site/content/docs/manifest/scheduled-job.en.md]:
```suggestion
The logging section contains log configuration parameters for your container's FireLens log driver.
```

**Review by huanjani — APPROVED** [review]
Thanks for discovering this hole in the docs and filling it so quickly!

File: site/content/docs/include/common-svc-fields.en.md (hunk 1 above)
Comment (Lou1415926) [reply site/content/docs/include/common-svc-fields.en.md]: done!

File: site/content/docs/include/common-svc-fields.en.md [hunk 3]
```diff
@@ -215,5 +215,25 @@ Optional. Defaults to `""`. The ID of the EFS access point to connect to. If usi
 
 <div class="separator"></div>
 
+<a id="logging" href="#logging" class="field">`logging`</a> <span class="type">Map</span>  
+The logging section contains log configuration parameters for your container's FireLens log driver. This will create a logging [sidecar](../developing/sidecars.en.md#sidecar-patterns).
```
Comment (iamhopaul123) [inline site/content/docs/include/common-svc-fields.en.md]: Maybe
```suggestion
The logging section contains parameters for log configuration options to send the log of the container using [Firelens](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_firelens.html) (see examples [here](../developing/sidecars.en.md#sidecar-patterns)).
```

File: site/content/docs/include/common-svc-fields.en.md [hunk 4]
```diff
@@ -215,5 +215,25 @@ Optional. Defaults to `""`. The ID of the EFS access point to connect to. If usi
...
+
+<span class="parent-field">logging.</span><a id="logging-image" href="#logging-image" class="field">`image`</a> <span class="type">Map</span>  
+Optional. The Fluent Bit image to use. Defaults to `amazon/aws-for-fluent-bit:latest`.
+
+<span class="parent-field">logging.</span><a id="logging-destination" href="#logging-destination" class="field">`destination`</a> <span class="type">Map</span>  
+Optional. The configuration options to send to the FireLens log driver.
+
+<span class="parent-field">logging.</span><a id="logging-enableMetadata" href="#logging-enableMetadata" class="field">`enableMetadata`</a> <span class="type">Map</span>  
+Optional. Whether to include ECS metadata in logs. Defaults to true.
```
Comment (iamhopaul123) [inline site/content/docs/include/common-svc-fields.en.md]:
```suggestion
Optional. Whether to include ECS metadata in logs. Defaults to `true`.
```

**Review by iamhopaul123 — COMMENTED** [review]
Yayy thank you so much for the quick turnaround on this!

**Review by iamhopaul123 — APPROVED** [review]
:shipit:

**Pull request merged**